src/videotile/DefaultVideoElementResolutionMonitor.ts

```diff
@@ -32,12 +32,15 @@ export default class DefaultVideoElementResolutionMonitor implements VideoElemen
     this.observerQueue.delete(observer);
   }
 
-  bindVideoElement(newElement: HTMLVideoElement): void {
-    this.element = newElement;
-    if (!this.element) {
-      this.resizeObserver.unobserve(this.element);
+  bindVideoElement(newElement: HTMLVideoElement | null): void {
+    if (!newElement) {
+      if (this.element) {
+        this.resizeObserver.unobserve(this.element);
+      }
+      this.element = newElement;
       return;
     }
+    this.element = newElement;
```

Comment:
Q: if `newElement` is null, is it supposed to be observed?

Reply:
If `newElement` it will be handled inside the `if (!newElement)` section above call `unobserve` if it was observing an element

Reply:
Q: I have limited context, if `this.element` and `newElement` are both not `null`, do we need to `unobserve` the object previously assigned to `this.element` before observe the new one?